src/plugins/tables/tables.js

```diff
@@ -114,11 +115,19 @@ var componentName = "wb-tables",
 						}
 					} );
 
-					// Add the container or the sorting icons
-					$elm.find( "th" ).append( "<span class='sorting-cnt'><span class='sorting-icons'></span></span>" );
+					// Handle sorting/ordering
+					var ordering = ( userDefaults && userDefaults.ordering === false ) ? false : true;
+					if ( ordering ) {
+						$elm.find( "th" ).each( function() {
+							var $th = $( this );
+							$th.html( "<button type='button' class='sorting-cnt' title='" + $th.html() + i18nText.aria.sortAscending + "'>" + $th.html() + " <span class='sorting-icons' aria-hidden='true'></span></button>" );
+							$th.removeAttr( "aria-label" );
+						} );
+						$elm.attr( "summary", "This table provides a sorting feature via the buttons across the column header row with only one instance visible at a time." );
```

Comment:
@GormFrank can you draft a French translation.

@thekodester We would might need to use an aria-label here considering the summary attribute is deprecated. For reference: https://html.spec.whatwg.org/multipage/tables.html#the-table-element and Obsolete - https://html.spec.whatwg.org/multipage/obsolete.html#HTMLTableElement-partial